app/views/declaration/exporter_details.scala.html

```diff
@@ -103,7 +103,7 @@
         @govukFieldset(Fieldset(
             attributes = Map("id" -> "details"),
                 legend = Some(Legend(
-                    content = Text(messages("")),
+                    content = Text(""),
```

Comment:
Also looks wrong - label with no text

Reply:
In line 110 you have `exportsName`

You can move exportsName content to this legend content and remove exportsName.

With this approach you get rid of empty message here

Reply:
Should I fix this bug as part of this ticket?  There was some other place with an empty label as well.

Reply:
I think it's not necessary, up to you :)

Reply:
I'll raise a new ticket to look at the two places where we have an empty label (from an accessibility point of view).

Could you take another look at my latest commit @PatrykRudnicki ?   This one removes redundant messages and fixes a couple of issues spotted on the way.